src/app/docs/inline-form/inline-form-docs.component.ts

```diff
@@ -1,11 +1,64 @@
 import {
-  ChangeDetectionStrategy,
+  ChangeDetectorRef,
   Component
 } from '@angular/core';
 
+import {
+  SkyDocsDemoControlPanelChange,
+  SkyDocsDemoControlPanelRadioChoice
+} from '@skyux/docs-tools';
+
+import {
+  SkyInlineFormButtonLayout,
+  SkyInlineFormCloseArgs,
+  SkyInlineFormConfig
+} from '../../public/public_api';
+
 @Component({
   selector: 'app-inline-form-docs',
-  templateUrl: './inline-form-docs.component.html',
-  changeDetection: ChangeDetectionStrategy.OnPush
+  templateUrl: './inline-form-docs.component.html'
 })
-export class InlineFormDocsComponent { }
+export class InlineFormDocsComponent {
+
+  public inlineFormButtonLayoutChoices: SkyDocsDemoControlPanelRadioChoice[] = [
+    { label: 'DoneCancel', value: SkyInlineFormButtonLayout.DoneCancel },
+    { label: 'DoneDeleteCancel', value: SkyInlineFormButtonLayout.DoneDeleteCancel },
+    { label: 'SaveCancel', value: SkyInlineFormButtonLayout.SaveCancel },
+    { label: 'SaveDeleteCancel', value: SkyInlineFormButtonLayout.SaveDeleteCancel }
+  ];
+
+  public demoSettings: {
+    inlineFormConfig: SkyInlineFormConfig;
+    showForm: boolean;
+    title: string;
+  } = {
+    inlineFormConfig: {
+      buttonLayout: SkyInlineFormButtonLayout.SaveCancel
+    },
+    showForm: false,
+    title: 'My inline form title'
```

Comment:
Superminor, but I'd delete `My` here and in the other instances and just go with `Inline form title`. (Not sure if we want a more real-world example given the bare bones nature of the demo.)

Also, this title shows up initially when I see the page, and then it shows up again as placeholder text in the field after clicking the initial link. It doesn't really make sense in the field, and it's not italicized like placeholder text is supposed to be. Can we either without placeholder text or display the placeholder text with the `placeholder` attribute on the input like we demo in the form classes docs?

Reply:
I reworked the demo to hopefully make more sense.